plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -313,11 +313,18 @@ static void build_command_line(struct ffmpeg_muxer *stream, struct dstr *cmd,
 		add_video_encoder_params(stream, cmd, vencoder);
 
 	if (num_tracks) {
-		dstr_cat(cmd, "aac ");
+		struct dstr name = {0};
+		dstr_init_copy(&name, obs_encoder_get_codec(aencoders[0]));
+		dstr_cat(&name, " ");
+		// Name needs to be lowercase
+		dstr_to_lower(&name);
+		dstr_cat_dstr(cmd, &name);
```

Comment:
Forcing lower case is not longer needed since #8090, right ?

Reply:
Yes, I have split this out into a separate PR #8496 to fix this issue even if this doesn't get merged yet. I'll rebase this PR later.